src/applications/avs/tests/components/YourHealthInformation.unit.spec.jsx

```diff
@@ -34,12 +34,15 @@ describe('Avs: Your Health Information', () => {
     expect(screen.getByTestId('recall-appointments').firstChild).to.have.text(
       'March 15, 2024TEST CLINIC (VETERANS LOCATION VIDEO )Clinic location: LOMA LINDA VA CLINIC',
     );
+    expect(screen.getByTestId('problems')).to.contain.text(
+      'Diabetes Last updated: June 10, 2023',
+    );
     expect(screen.getByTestId('smoking-status')).to.have.text('Current smoker');
     expect(screen.getByTestId('immunizations')).to.contain.text(
       'COVID-19 (PFIZER)',
     );
     expect(screen.getByTestId('allergies-reactions')).to.contain.text(
-      'SIMVASTATIN',
+      'SIMVASTATINVerified date: June 10, 1999Severity: None notedReaction: myopathy',

```

Comment:
Is there a reason this text is so squashed together?

Reply:
That's a good question. I had assumed it was just an artifact of the testing framework, but it may be an issue with the markup. I'll add an issue to follow up on that for AVS.